doc/api/url.md

```diff
@@ -859,11 +859,15 @@ new URLSearchParams([
 
 Append a new name-value pair to the query string.
 
-#### `urlSearchParams.delete(name)`
+#### `urlSearchParams.delete(name, value)`
 
```

Comment:
@anonrig @debadree25 
Made the doc changes.

Had couple of doubts.

i) how to show that `value` is optional argument. 
ii) Do I need to mention something like "added in version x.x.x". I saw something mentioned [here](https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md#step-3-code)

<!-- YAML
added: REPLACEME
-->

Reply:
i) to show wrap it in [ something like ```urlSearchParams.delete(name[, value])```
ii) Yes